### ISSUE TITLE
[Enhancement]: Create notice templates for warning and default block notices

### ISSUE THREAD

**Issue by ralucaStan** [woothemes/woocommerce#44324]
### Describe the solution you'd like

After #44283 the block notices will be a default for the block themes.

In addition to the classic notices (info, error, success), the block notices introduce 2 new notice types: warning and default. see https://github.com/woocommerce/woocommerce/blob/16f2f9024a7f0b29b6496b7ad76cb1e1bc2ab7e2/plugins/woocommerce-blocks/assets/js/base/components/notice-banner/index.tsx#L23

These two 2 types don't have notice templates that people can use to modify them. 

### Additional context

These new notice templates should only be used by the block themes and shouldn't return anything in a classic theme.

**Comment by kkoteen** [comment]
Checked against current trunk: notice templates still only cover error / notice / success. warning and default are JS-only (the notice-banner component in cart/checkout) — server-rendered notices never use them, so warning.php / default.php would be orphaned templates with no caller. default is already covered by notice.php.

Closing as won't-do — easy to revisit if server-side warning notices ever become a real need. 

@ralucaStan please reopen if you still think these would be helpful.
